docker/supervisord.conf

```diff
@@ -30,7 +27,6 @@ autorestart=true
 
 [program:queue-worker]
 command=/usr/local/bin/php /var/www/html/artisan queue:work --tries=3
-user=www-data
```

Comment:
I'm assuming you removed this because you intend supervisord to run as www-data? if not then add this back

Reply:
Everything in the container is non-root, and it's www-data in the current Dockerfile, however if the user is ever changed for whatever reason (maybe you want it to be pelican? I don't know) supervisord will get mad, so I figured it's pointless to keep it anyways if it's running as non-root anyways.